building/building_index.adoc

```diff
@@ -62,6 +62,141 @@ This will build the image which could then be pushed to a registry location. Don
+[Unit]
+Description=Service for virtual machines hosted on VMware
+Documentation=http://github.com/vmware/open-vm-tools
+ConditionVirtualization=vmware
+
+[Service]
+ExecStartPre=/bin/bash -c 'systemctl import-environment'
+ExecStartPre=/bin/bash -c 'export -p > /tmp/open-vm-tools-bash-env'
+ExecStart=$EXEC_START
+ExecStop=$EXEC_STOP
+WorkingDirectory=$DESTDIR
+
+[Install]
+WantedBy=multi-user.target
+----
+
+The config.json.template is similar to the Docker run label or line. Below shows the environment variables, bind mounts and the command to execute "/usr/bin/init.sh"
+
+[source,shell]
+----
+{
+    "ociVersion": "1.0.0",
+    "platform": {
+        "os": "linux",
+        "arch": "amd64"
+    },
+    "process": {
+        "terminal": false,
+        "user": {},
+        "args": [
+            "/usr/bin/init.sh"
+        ],
+        "env": [
+            "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+            "TERM=xterm",
+            "NAME=open-vm-tools",
+            "SYSTEMD_IGNORE_CHROOT=1"
+        ],
+...
+omitted
+    "mounts": [
+            {
+            "destination": "/run/systemd",
+            "type": "bind",
+            "source": "/run/systemd",
+            "options": [
+                "rw",
+                "rbind",
+                "rprivate"
+            ]
+        },
+...
+omitted
+----
+
+Often times, executing a single command via the container is not enough. The above command init.sh stages the container environment and ensures both VGAuthService and vmtoolsd is executed inside the container.
+
+[source,shell]
+----
+#!/bin/sh
+source /tmp/open-vm-tools-bash-env
+
+COMMAND=/usr/local/bin/vmware-toolbox-cmd
+if [ ! -e $COMMAND ]
+  then
+    echo 'runc exec -t open-vm-tools vmware-toolbox-cmd "$@"' > /usr/local/bin/vmware-toolbox-cmd
+    chmod +x /usr/local/bin/vmware-toolbox-cmd
+fi
+exec /usr/bin/VGAuthService -s &
+exec /usr/bin/vmtoolsd 
+----
+
+Here are the commands to execute via the atomic CLI to install and convert a system container. Provided we have already built the open-vm-tools container from the Dockerfile listed above. 
+
+[source,shell]
+----
+atomic pull --storage=ostree docker:open-vm-tools
+atomic install --system open-vm-tools
+systemctl start open-vm-tools
+----
+
+Similarly, we can pull this container from the Red Hat registry and install it in the same fashion.
+[source,shell]
+----
+atomic pull --storage ostree registry.access.redhat.com/rhel7/open-vm-tools
+atomic install --system registry.access.redhat.com/rhel7/open-vm-tools
+systemctl start open-vm-tools 
+----
+
+The atomic install command installs the systemd unit file from the container from its /exports/ directory and sets the service to enable. The systemctl command below that starts the service immediately instead of awaiting a reboot. 
+
+More examples of system containers can be found [https://github.com/projectatomic/atomic-system-containers]here. This includes the open-vm-tools example for CentOS. 
+
```

Comment:
The URL should be done the other way around, otherwise it doesn't render :)

https://github.com/projectatomic/atomic-system-containers[here]